ps_linklist.php

```diff
@@ -99,7 +99,7 @@ public function __construct()
         $this->displayName = $this->trans('Link List', [], 'Modules.Linklist.Admin');
         $this->description = $this->trans('Give more visibility to your content/static pages (CMS, external pages, or else), where you want and when you want, to make your visitors feel like shopping on your store.', [], 'Modules.Linklist.Admin');
 
-        $this->ps_versions_compliancy = ['min' => '1.7.8.0', 'max' => _PS_VERSION_];
+        $this->ps_versions_compliancy = ['min' => '8.1.0', 'max' => _PS_VERSION_];
```

Comment:
I'm not sure I understand why the minimal PS Version for linklist 5.1 would be PS 8.1 and not 8.0 for example, could you explain it please ?

Reply:
Because the `TranslatorAwareTrait` force the Symfony Contracts (and not anymore Symfony Component... which is a BC Break... in a minor version). (PR PrestaShop/PrestaShop#30496)

The `8.1` tag is not existing in Docker :whale:.

Reply:
Thank you but then I think you should edit the PR description as I find it confusing, that's why I commented in the first place because you talked about 8.0.0 😛

Reply:
Oops, fixed. Thanks @0x346e3730